src/output.cpp

```diff
@@ -1958,8 +1958,9 @@ void play_music(std::string)
 {
 }
 
-void play_sound(std::string)
+void play_sound_effect(std::string id, std::string variant, int volume)
 {
+    (void)id;(void)variant;(void)volume;
```

Comment:
The easier way of avoid the "unused parameter" warning is to comment the parameter name int the function header:
```C++
void play_sound_effect(std::string /*id*/, std::string /*variant*/, int /*volume*/)
```

Those `(void)id;` lines tend to stay in even after the function has been changed to use the parameter.